README.md

```diff
@@ -750,14 +753,30 @@ without calling SLO. The mod_auth_mellon cookie session will be
 invalidated and the session will be removed from the mod_auth_mellon cache. 
 SLO will not be possible after the mod_auth_mellon session is invalidated.
 If this functionality is enabled, invalidate the session by calling 
-the endpoint "<endpoint path>/invalidate".
+the endpoint "<endpoint path>/invalidate?ReturnTo=<valid url>".
+ReturnTo parameter is required.
+
 Here is a sample configuration to enabled this feature:
+
 ```ApacheConf
 MellonEnabledInvalidateSessionEndpoint On
 ```
 Default value is Off
 
 
+## Disabled curl Expect Logic
+Disabling the Expect Logic saves an additional network round-trip and is thus
+a good idea when the request isn't extremely large and the probability for rejection is low.
+For some Apache server version, the Expect Header is not properly managed and the curl command will 
+wait for 1 sec. before sending the body of the request. This will improve performance with 
+the HTTP-Artifact binding.
+
+```ApacheConf
+MellonDisabledCurlExpectLogic On
```

Comment:
So the way this option is phrased is confusing.
Firs t of all if I understand this right, while this behavior is to improve Curl's situation, it is in no way limited to interactions with Curl, so mentioning Curl int he option name makes little sense to me (it is totally ok to mention Curl in the docs text that explain why it is being done).

Secondly "" Disable = On "" is alays going to confuse people as to what is  going to happen.

I would change the option to be named:
MellonSendExpectHeader Off

It will be On by default, which means it will send the Expect Header as done now, but can be turn off for the curl case.